rubygems/rubygems.org
# Add rubygem download ranking

This PR adds a ranking based on downloads for rubygems and displays it on the rubygems page, below the amount of total downloads. It caches the result once, and refreshes it every 24 hours, if the rubygem is not ranked yet (i.e. is new), it will refresh the ranking.

# Testing
1. Go to a gems show page
1. See the rank below the amount of downloads

<img width="1163" alt="Screen Shot 2023-05-20 at 21 33 00" src="https://github.com/rubygems/rubygems.org/assets/4212649/7a625895-8c3e-4e9f-a8b8-36eeef66cc6e">

<img width="1156" alt="Screen Shot 2023-05-20 at 21 32 46" src="https://github.com/rubygems/rubygems.org/assets/4212649/245f1a85-c125-4ee6-96ca-902c66eeec65">

---

Closes https://github.com/rubygems/rubygems.org/issues/3658
